android/src/main/kotlin/com/github/fluttercommunity/flutterdownloader/DownloadWorker.kt

```diff
@@ -0,0 +1,187 @@
+package com.github.fluttercommunity.flutterdownloader
+
+import android.content.Context
+import android.net.Uri
+import android.util.Log
+import androidx.work.CoroutineWorker
+import androidx.work.WorkerParameters
+import kotlinx.coroutines.*
+import java.io.BufferedInputStream
+import java.io.FileOutputStream
+import java.net.HttpURLConnection
+import java.net.SocketException
+import java.net.URL
+import java.net.URLDecoder
+import java.util.*
+import java.util.regex.Pattern
+import kotlin.coroutines.cancellation.CancellationException
+
+/***
+ * A simple worker that will post your input back to your Flutter application.
+ *
+ * It will block the background thread until a value of either true or false is received back from Flutter code.
```

Comment:
```suggestion
 * A simple worker that posts the input its given back to the Flutter application.
 *
 * It blocks the background thread until a value of either true or false is received back from Flutter code.
```

I'd appreciate some more explaining here: what is that _true or false value_ for? When is it "received back from Flutter code"?

[Also, I like when software documentation is written in Present Simple](https://en.wikiversity.org/wiki/Technical_writing/Style#Understanding_present_tense).

Reply:
I need to improve the documentation. That part is copied from somewhere else I'll change it